content/docs/operator/upgrade-vault.md

```diff
@@ -7,5 +7,5 @@ weight: 50
 To upgrade Vault using the Bank-Vaults operator, complete the following steps.
 
 1. Check the release notes of Vault for any special upgrade instructions. Usually there are no instructions, but it's better to be safe than sorry.
-1. [Adjust the spec.image in field in the Vault custom resource](https://github.com/bank-vaults/bank-vaults/blob/e83a577ddc5fc36f1756d17d8ff63a8ad02438f3/operator/deploy/cr.yaml#L7).
+1. [Adjust the spec.image in field in the Vault custom resource](https://github.com/bank-vaults/vault-operator/blob/main/deploy/cr.yaml#L7).
```

Comment:
This does not work due to recent changes to vault-operator repo. Would also be good to use specific commit when referencing lines rather than branch since the line will always reference the same value.